plugins/inputs/slab/README.md

```diff
@@ -0,0 +1,53 @@
+# Slab Input Plugin
+
+This plugin collects details on how much memory each entry in Slab cache is
+consuming. For example, it collects the consumption of `kmalloc-1024` and
+`xfs_inode`. Since this information is obtained by parsing `/proc/slabinfo`
+file, only Linux is supported. The specification of `/proc/slabinfo` has
+not changed since [Linux v2.6.12 (April 2005)](https://github.com/torvalds/linux/blob/1da177e4/mm/slab.c#L2848-L2861),
+so it can be regarded as sufficiently stable. The memory usage is
+equivalent to the `CACHE_SIZE` column of `slabtop` command.
+If the HOST_PROC environment variable is set, Telegraf will use its value instead of `/proc`
+
+**Note: `/proc/slabinfo` is usually restricted to read as root user. Make sure telegraf can execute `sudo` without password.**
+
+## Configuration
+
+```toml
+# Get slab statistics from procfs
+[[inputs.slab]]
+  # no configuration
```

Comment:
Hey - Dave and I chatted about this some more offline. If a user is only looking at the config, they may never see the recommendation about the sudoers file change in the readme. We wanted to directly people to the README to ensure that they lock down the sudoers file.  Can we add the following:

```suggestion
  # no configuration - please see the plugin's README for steps to configure sudo properly
```